.husky/pre-commit

```diff
@@ -1,4 +1,31 @@
 #!/bin/sh
 . "$(dirname "$0")/_/husky.sh"
 
+echo "Running pre-commit checks..."
+
+# Run the link checker
+echo "Checking links in HTML files..."
+node check-links.js
+LINK_CHECK_EXIT_CODE=$?
+
+# Run lint-staged
+echo "Running lint-staged..."
```

Comment:
I don't think this is the right place for this code, we usually run the check/linting commands using lint-staged to apply only to staged files and then use husky to run lint-staged pre-commit.